Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ironoxide-java"
-version = "0.3.0"
+version = "0.5.0"
```

Comment:
This is to sync the Rust and Scala versions to 0.5?

Reply:
Correct